acceptance/pipelines/run/basic/script

```diff
@@ -0,0 +1,6 @@
+trace $CLI bundle deploy
+title "run pipeline"
+trace errcode $PIPELINES run my_pipeline
+rm -f out.requests.txt
+# This printf is here to fix the whitespace linter error (ensures exactly one newline at end)
+printf "<EOL>\n"
```

Comment:
Do we need this? I see that errcode helper always prints new lines. There also might be something wrong with our implementation